docs/guides/core-concepts/writing-and-organizing-tests.mdx

```diff
@@ -292,7 +292,18 @@ is set to look for one of the following files:
 
 :::
 
-::testing-type-specific-option{option=supportFile}
+:::info
+
+<strong><Icon name="cogs"></Icon> supportFile per testing
+type</strong>
+
+Depending on which [testing type](/guides/core-concepts/testing-types)
+you are using, you can configure yoursupportFile accordingly.
+
+- [component](/guides/references/configuration#component)
+- [e2e](/guides/references/configuration#e2e)
+
```

Comment:
Couple of tweaks. I'm not sure if we'd need to capitalize the types in this context but I believe that's an initiative on the PMM side. 

```suggestion
you are using, you can configure your `supportFile` accordingly.

- [Component](/guides/references/configuration#component)
- [E2E](/guides/references/configuration#e2e)

```